### DISCUSSION TITLE
Работа игр

### DISCUSSION THREAD

**Discussion by AlexOlmer** [Flowseal/zapret-discord-youtube#367] [General]
При использовании обхода блокировки Discord (версия 1.2, 1.3, 1.5.1.) невозможно стало играть в Call of Duty
Игра запускается, но вылетает без ошибок. Вылетает на любой стадии.
Помогает с запуском только выключение сервиса service_remove.bat и выполнение команд
ipconfig /flushdns
netsh winsock reset

Есть ли возможность решить эту проблему?

**Comment by Flowseal** [comment]
https://github.com/Flowseal/zapret-discord-youtube/issues/310
